lib/queue/consumer.ts

```diff
@@ -225,3 +242,26 @@ export class Consumer implements QueueConsumer {
 		return eventContract;
 	}
 }
+function graphileLoggerToJellyfishLogger(logContext: LogContext) {
```

Comment:
Any reason to have this out in its own function and not inline when setting up `this.graphileRunner`?

Reply:
I think it's simpler to make the initialization have only one line that sets up the logger:
```
   logger: new Logger(graphileLoggerToJellyfishLogger(logContext)),
```

and create this mapping function elsewhere.

As I commented above this indirection level that `graphile` has doesn't make things easy. Anyway, I'll give it another go and try to make things simpler.

Reply:
Either way works, I just normally don't like defining functions/variables that are only ever referenced once.